packages/web3/test/stress/many_requests/index.html

```diff
@@ -0,0 +1,421 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/web3/4.2.2/web3.min.js"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/ganache@7.3.2/dist/web/ganache.min.js"></script>
```

Comment:
The problem is a cors in a browser. I can’t use something different. I only find to run ganache in browser. In this case there is no cors problems

Reply:
This is a good alternative to using Ganache, it is `@ethereumjs/vm`. And here is an example of using it: https://github.com/ethereumjs/ethereumjs-monorepo/blob/e2803ab1ed90f7fbcfada059829011fc20da27e1/packages/vm/examples/browser.html
I you face issues using this easily. I think that you may leave this code as is and create a followup issue. But I think it would be simple as Ganache and Remix depends on it.